scripts/set-priorities

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
+
+import polib
+import sys
+
+def main (f):
+    po = polib.pofile(f, wrapwidth=0)
+    priorities = [('en/git-init.txt','300'),('en/git-clone.txt','300'),('en/urls.txt','300'),('en/git-add.txt','280'),('en/git-status.txt','280'),('en/git-diff.txt','280'),('en/diff-options.txt','280'),('en/diff-format.txt','280'),('en/git-commit.txt','280'),('en/date-formats.txt','280'),('en/i18n.txt','280'),('en/git-reset.txt','280'),('en/git-rm.txt','280'),('en/git-mv.txt','280'),('en/git-show.txt','260'),('en/pretty-options.txt','260'),('en/pretty-formats.txt','260'),('en/diff-generate-patch.txt','260'),('en/git-log.txt','260'),('en/line-range-format','260'),('en/rev-list-options.txt','260'),('en/git-shortlog.txt','260'),('en/mailmap.txt','260'),('en/git-describe.txt','260'),('en/git-branch.txt','240'),('en/git-checkout.txt','240'),('en/git-merge.txt','240'),('en/merge-options.txt','240'),('en/merge-strategies.txt','240'),('en/config/merge.txt','240'),('en/config/fmt-merge-msg.txt','240'),('en/mergetools-merge.txt','240'),('en/git-mergetool.txt','240'),('en/git-stash.txt','240'),('en/git-tag.txt','240'),('en/git-worktree.txt','240'),('en/git-fetch.txt','220'),('en/fetch-options.txt','220'),('en/pull-fetch-param.txt','220'),('en/urls-remotes.txt','220'),('en/transfer-data-leaks.txt','220'),('en/git-pull.txt','220'),('en/git-push.txt','220'),('en/git-remote.txt','220'),('en/git-submodule.txt','220')]
+    for entry in po:
+        b = False
+        for flag in entry.flags:
+            if flag.startswith("priority"):
+                entry.flags.remove(flag)
+                break
+        for i in range(len(priorities)):
+            for j in range(len(entry.occurrences)):
```

Comment:
iterate directly over the entries, instead of using indexes.

Reply:
I went with the index based approach because I hadn't thought of the

````
                                    break
		            else:
		                continue
		            break
````
construct yet. I've tried to cut the loops short by setting i and j to their respective len(...).